src/Logging/Connection.php

```diff
@@ -26,8 +20,9 @@ final class Connection implements ServerInfoAwareConnection
      */
     public function __construct(ConnectionInterface $connection, LoggerInterface $logger)
     {
-        $this->connection = $connection;
-        $this->logger     = $logger;
+        $this->logger = $logger;
+
+        parent::__construct($connection);
```

Comment:
In this case, it's purely a matter of taste, but for instance, Java requires that the parent constructor is called in the first statement. Would it make sense to keep it this way here as well?

Reply:
It doesn't make a difference here, I suppose. I'll change it.